src/main/java/uk/gov/hmcts/ccd/data/casedetails/JurisdictionMapper.java

```diff
@@ -13,7 +13,7 @@ public JurisdictionDisplayProperties toResponse(Jurisdiction jurisdiction) {
         result.setId(jurisdiction.getId());
```

Comment:
shall we rename Jurisdiction to JurisdictionDefinition?

Reply:
Missed it... Now Renamed to JurisdictionDefinition